doc/sphinx/compiling.rst

```diff
@@ -32,69 +33,267 @@ Installation Prerequisites
+  used as well.
+
 * If you want to build the Matlab toolbox and you have a 64-bit copy of
   Windows, by default you will be using a 64-bit copy of Matlab, and therefore
   you need to compile Cantera in 64-bit mode. For simplicity, it is highly
   recommended that you use a 64-bit version of Python to handle this
-  automatically.
+  automatically. Note that the default download from the Python website is
+  for a 32-bit installer, and you will need to select the 64-bit installer
+  specifically.
 
-* It is generally helpful to have SCons and Python in your PATH. This can
-  usually be accomplished by adding the top-level Python directory
-  (e.g. ``C:\Python27``) to your PATH. This is accessible from::
+* It is generally helpful to have SCons and Python in your ``PATH`` environment
+  variable. This can be done by checking the appropriate box during the
+  installation of Python or can be accomplished by adding the top-level Python
+  directory and the ``Scripts`` subdirectory (e.g.,
+  ``C:\Python27;C:\Python27\Scripts``) to your ``PATH``. The dialog to change
+  the ``PATH`` is accessible from::
 
       Control Panel > System and Security > System > Advanced System Settings > Environment Variables
+
+  Make sure that the installation of Python that has SCons comes first on your
+  ``PATH``.
+
 * In order to use SCons to install Cantera to a system folder (e.g. ``C:\Program
   Files\Cantera``) you must run the ``scons install`` command in a command
-  prompt that has been launched by selecting the *run as administrator* option.
+  prompt that has been launched by selecting the *Run as Administrator* option.
+
+.. _sec-windows-reqs:
+
+Windows Requirements
+^^^^^^^^^^^^^^^^^^^^^^^
+
+* Windows 7 or later; either 32-bit or 64-bit
+
+* To build any of the Cantera modules, you will need to install
+
+  * Python 2.7
+
+    * https://www.python.org/downloads/
+
+    * Be sure to choose the appropriate architecture for your system - either
+      32-bit or 64-bit
+
+    * When installing, make sure to choose the option to add to your ``PATH``
+
+  * SCons
+
+    * https://pypi.python.org/pypi/SCons
+
+    * Be sure to choose the appropriate architecture for your system - either
+      32-bit or 64-bit
+
+  * One of the following supported compilers
+
+    * Microsoft compilers
+
+      * https://www.visualstudio.com/downloads/
+
+      * Known to work with Visual Studio 2013 (MSVC 12.0) and Visual Studio 2015
+        (MSVC 14.0)
+
+    * MinGW compilers
+
+      * http://mingw-w64.org/
+
+      * Known to work with Mingw-w64 3.0, which provides GCC 4.8. Expected to
+        work with any version that provides a supported version of GCC and
+        includes C++11 thread support.
+
+      * The version of MinGW from http://www.mingw.org/ is 32-bit only and
+        therefore cannot be used to build a 64-bit module. Versions of
+        MinGW that provide a 64-bit compiler are available from
+        http://mingw-w64.org/
```

Comment:
Yes, I think we should specifically say not to use the mingw.org builds. To add to the confusion mingw-w64 *also* provides 32-bit builds that should work. 😕